src/devhostagent.restorationjob/Dockerfile

```diff
@@ -20,15 +20,17 @@ ENV MINDARO_BUILD_NUMBER=${MindaroBuildNumber}
 RUN dotnet publish -c ${Configuration} -o /output devhostAgent.restorationjob.csproj
 
 # Final container
-FROM mcr.microsoft.com/dotnet/aspnet:6.0-alpine3.16 as final
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-cbl-mariner2.0 as final
```

Comment:
Can we try distroless image ```FROM mcr.microsoft.com/dotnet/runtime-deps:6.0-cbl-mariner2.0-distroless as final ``` ? which is more performant and anyway we are installing the OS dependencies, tools manually using tndf package manager. This is just a suggestion if this causes any problems, we can always revert back.

Reply:
distroless is missing the dependencies installed on 26-29, unless we don't need those?

Reply:
I thought distroless got those dependencies covered. if that's the case we can resolve this. :), Thanks.